examples/app/fourier_animated.py

```diff
@@ -154,7 +152,7 @@ def create_centric_plot(foos, title='', r = 1, y_range=(-2, 2), period = pi/2, c
 cf2 = lambda x: (4*np.cos(3*x))/(3*pi)
 cf3 = lambda x: (4*np.cos(5*x))/(5*pi)
 cf4 = lambda x: (4*np.cos(7*x))/(7*pi)
-fourier = OrderedDict(
+fourier = dict(
```

Comment:
I'll revert the change. I tested the examples and both worked correctly. But this could've just been a coincidence of key/value ordering.